packages/itwin/grouping-mapping-widget/src/test/QueryBuilder.testdata.json

```diff
@@ -0,0 +1,808 @@
+            ],
+            "type": {
+              "typeName": "string"
+            },
+            "parent": {
+              "pathToPrimaryClass": [
+                {
+                  "sourceClassInfo": {
+                    "id": "id1",
+                    "name": "A:B",
+                    "label": "AB label"
+                  },
+                  "targetClassInfo": {
+                    "id": "id2",
+                    "name": "A:C",
+                    "label": "AC label"
+                  },
+                  "isPolymorphicTargetClass": true,
+                  "relationshipInfo": {
+                    "id": "id3",
+                    "name": "A:D",
+                    "label": "AD label"
+                  },
+                  "isPolymorphicRelationship": true,
+                  "isForwardRelationship": false
+                },
+                {
+                  "sourceClassInfo": {
+                    "id": "id2",
+                    "name": "A:C",
+                    "label": "AC label"
+                  },
+                  "targetClassInfo": {
+                    "id": "id4",
+                    "name": "Z:X",
+                    "label": "ZX label"
+                  },
+                  "isPolymorphicTargetClass": false,
+                  "relationshipInfo": {
+                    "id": "id5",
+                    "name": "A:E",
+                    "label": "AE label"
+                  },
+                  "isPolymorphicRelationship": true,
+                  "isForwardRelationship": true
+                }
+              ],
+              "contentClassInfo": {
+                "name": "A:B"
+              }
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "when no properties added, return empty string",
+      "expectedResult": "",
+      "operations": []
+    },
+    {
+      "name": "when property is float, return query string with ROUND",
+      "expectedResult": "SELECT A.B.ECInstanceId FROM A.B WHERE ROUND(A.B.somePropName, 4)=3.1416",
+      "operations": [
+        {
+          "expectedResult": true,
+          "operationType": "addProperty",
+          "propertyRecord": {
+            "value": {
+              "valueFormat": 0,
+              "value": 3.14159
+            },
+            "property": {
+              "typename": "notNavigation"
+            }
+          },
+          "propertiesField": {
+            "properties": [
+              {
+                "property": {
+                  "name": "somePropName",
+                  "classInfo": {
+                    "name": "A:B"
+                  }
+                }
+              }
+            ],
+            "type": {
+              "typeName": "number"
+            },
+            "parent": {
+              "pathToPrimaryClass": []
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "when property needsQuotes, return query string with property value in quotes",
```

Comment:
Something is wrong with this test. There are no quotes.

Reply:
confirmed